src/devhub/devhub.js

```diff
@@ -109,42 +109,53 @@ async function mainSeeds() {
 
     if (!include) continue;
 
-    const seedDuration = formatDuration(
-      (record.seed_timestamp_end - record.seed_timestamp_start) * 1000,
-    );
-    const seedFreshness = formatDuration(
-      Date.now() - (record.seed_timestamp_start * 1000),
-    );
-    const rowDom = document.createElement("tr");
-
-    const seedSuccess = record.fuzzer === "canary" ? !record.ok : record.ok;
-    if (seedSuccess) {
-      rowDom.classList.add("success");
+    const seed_duration_ms = (record.seed_timestamp_end - record.seed_timestamp_start) * 1000;
+    const seed_freshness_ms = Date.now() - (record.seed_timestamp_start * 1000);
+    const staleness_threshold_ms = 2 * 60 * 60 * 1000;
+    const canery_is_stale = record.fuzzer === "canary" &&
+      !pullRequestNumber(record) &&
+      seed_freshness_ms > staleness_threshold_ms;
+
+    const row_dom = document.createElement("tr");
+
+    let seed_success = record.fuzzer === "canary" ? !record.ok : record.ok;
+    if (canery_is_stale) seed_success = false;
+    if (seed_success) {
+      row_dom.classList.add("success");
     } else {
       seedFailCount++;
     }
     if (record.commit_sha != commit_previous) {
       commit_previous = record.commit_sha;
-      rowDom.classList.add("group-start");
+      row_dom.classList.add("group-start");
```

Comment:
👍 

Might rename others snake case while we are at it?

Reply:
Everything 🐍?

Reply:
Yup!

Reply:
Sorry for not getting casing consistently initially!

Reply:
No worries! I thought we would incrementally switch to snake case.